vignettes/get_commits_stats.Rmd

```diff
@@ -0,0 +1,84 @@
+---
+title: "Get commits stats"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Get commits stats}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+```{r, include = FALSE}
+knitr::opts_chunk$set(
+  collapse = TRUE,
+  comment = "#>",
+  fig.width = 7,
+  fig.height = 4
+)
+```
+
+When pulling commits you can choose to scan only specific repositories (not whole organizations). If that is your wish, pass your repositories (fullnames, i.e. `org/repository` path) to the `repos` parameter instead of `orgs` parameter.
+
+> Example workflow makes use of public GitLab, but is rather more plausible, that you will use your internal git platforms.
+
+```{r}
+library(GitStats)
```

Comment:
vigniettes do not build properly:
`Quitting from lines 24-37 (get_commits_stats.Rmd) 
Error in library(GitStats) : there is no package called 'GitStats'
Calls: <Anonymous> ... withVisible -> eval_with_user_handlers -> eval -> eval -> library
In addition: Warning message:
In do_once((if (is_R_CMD_check()) stop else warning)("The function xfun::isFALSE() will be deprecated in the future. Please ",  :
  The function xfun::isFALSE() will be deprecated in the future. Please consider using base::isFALSE(x) or identical(x, FALSE) instead.
Execution halted`